sphinxcontrib/needs/filter_common.py

```diff
@@ -58,6 +58,14 @@ def collect_filter_attributes(self):
         return collected_filter_options
 
 
+def load_needs_extra_filter_data(app, doctree, fromdocname):
+    """
+    Load needs external filter data from conf.py and set as global variable.
+    """
+    global NEEDS_EXTERNAL_FILTER_DATA
```

Comment:
NEEDS_EXTERNAL_FILTER_DATA should be defined on module level, if global

Reply:
Done.